forge/containers/wrapper.js

```diff
@@ -241,9 +190,10 @@ module.exports = {
         }
     },
     restartFlows: async (project, options) => {
-        if (this._isBillingEnabled()) {
-            await this._subscriptionHandler.requireTrialProjectOrActiveSubscription(project)
-        }
+        // This will perform all checks needed to ensure this instance
+        // can be started - throws err if not
+        await project.Team.checkInstanceStartAllowed(project)
```

Comment:
That this needs a comment to explain how the method works each time feels like a code smell to me.

How about naming it `ensureInstanceStartAllowed`, or `throwIfInstanceStartNotAllowed`?